### PR TITLE
Update glossary entry for Quantity #30885

### PR DESCRIPTION
**Pull request by KhushPatibandha**
Issue - #30885 
This is my first PR, do let me know if there are any changes required.
for reference "https://github.com/kubernetes/website/pull/39856" this is my previous PR that failed some checks, so I closed that and create a new one.

### CODE REVIEW

**Comment by KhushPatibandha** [comment]
@NitishKumar06 checks are done on this one right? it's just waiting for the review and I did apply the bullet point changes that you pointed out, if there is anything else to correct do let me know.

**Comment by KhushPatibandha** [comment]
> > @NitishKumar06 checks are done on this one right? it's just waiting for the review and I did apply the bullet point changes that you pointed out, if there is anything else to correct do let me know.
> 
> Please mention the issue this PR is trying to solve - In case this is a new PR, do mention the old PR so that reviewers can take some reference from the old PR if required.

I mentioned the issue and also tagged the previous PR which I closed for reference, if there is anything do let me know.

**Review by Arhell — COMMENTED** [review]
/easycla

**Comment by KhushPatibandha** [comment]
> /easycla

Hey, I had no idea I have to sign some agreement but I have done it now.

**Review by NitishKumar06 — COMMENTED** [review]
PTAL

File: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 1]
```diff
@@ -21,6 +21,14 @@ When you create a Pod, you can set environment variables for the containers
...
 `envFrom` field in the configuration file.
 
+Now lets look at the difference between `env` and `envFrom`
+
+`env` : This is a field in the pod specification that allows you to set environment variables for a specific container.
+
+`envFrom` :  This is a field in the pod specification that allows you to set environment variables by referencing a ConfigMap or a Secret. When you use `envFrom`, all the key-value pairs in the referenced ConfigMap or Secret are set as environment variables in the container. Hence `envFrom` is used to set multiple environment variables by referencing a ConfigMap or Secret
+
+You can read more about <a href="https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables"> ConfigMap </a> and <a href="https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#configure-all-key-value-pairs-in-a-secret-as-container-environment-variables"> Secret </a>      
+
```
[the remark this answers is not in the document]
Comment (KhushPatibandha) [reply content/en/docs/tasks/inject-data-application/define-environment-variable-container.md:31]: Yes, my bad I was working on another issue today and it happened by mistake, I'll fix it asap

[the remark this answers is not in the document]
Comment (NitishKumar06) [reply content/en/docs/tasks/inject-data-application/define-environment-variable-container.md:31]: Sure! Please take your time.

**Comment by KhushPatibandha** [comment]
@NitishKumar06 I did resolve that issue.

**Pull request closed without merging**